plugins/auth-backend/config.d.ts

```diff
@@ -185,6 +185,11 @@ export interface Config {
       /** @visibility frontend */
       cfaccess?: {
         teamName: string;
+        /** @visibility secret */
```

Comment:
Hey it's actually really important to set this to deep visibility

```suggestion
        /** @deepVisibility secret */
```

so that it bubbles down

Reply:
I didn't know that was possible! Thanks.